goncurses.c

```diff
@@ -88,3 +88,13 @@ int ncurses_untouchwin(WINDOW *win) { return untouchwin(win); }
 int ncurses_wattrset(WINDOW *win, int attr) { return wattrset(win, attr); }
 int ncurses_wstandend(WINDOW *win) { return wstandend(win); }
 int ncurses_wstandout(WINDOW *win) { return wstandout(win); }
+
+bool goncurses_set_escdelay(int size) {
+  bool code = true;
```

Comment:
It's not necessary to use a temp variable for something like this. You can just `return bool` from inside the `#ifdef` block. Do the same within the `#else` block for returning true. Everything else looks good.

Reply:
OK. Thank you.

I fixed. Please confirm.